application/config.json.template

```diff
@@ -45,5 +45,43 @@
        ],
        "categoryRoleSuffix": " - Helper"
    },
-    "mediaOnlyChannelPattern": "memes"
+    "mediaOnlyChannelPattern": "memes",
+    "blacklistedFileExtension": [
+        "application",
+        "bat",
+        "cmd",
+        "com",
+        "cpl",
+        "exe",
+        "gadget",
+        "hta",
+        "inf"
```

Comment:
Missing a commah

Reply:
already fixed in #553

Reply:
weird. I cloned the develop branch, it there wasn't the comma

Reply:
> weird. I cloned the develop branch, and there wasn't a comma

update ur local folder